### ISSUE TITLE
Accessibility: Some scrollable regions don't have keyboard access

### ISSUE THREAD

**Issue by bedmison** [web-cat/code-workout#241]
All scrollable regions should have keyboard access. This issue shows up in all of the CodeMirror controls that are scrollable (div Codemirror-scroll)

---

From the WCAG 2.1 spec:

To solve this problem, you need to fix at least (1) of the following:

- Element should have focusable content
- Element should be focusable

**Comment by nelsonisaac** [comment]
Hello @bedmison ,

Can you assign this task to me please. I'd like to work on this.

Thanks
